language-design.md

```diff
@@ -69,7 +155,20 @@ The Yang and Rabkin article talks mainly about general-purpose languages. In
 what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
-
+One thing we believe is that DSLs are just as susceptible as general-purpose
+programming languages to the biases and stereotypes people impose. DSLs, by
+design, are not as big and popular as other languages such as Java and Python.
+Because of this, they might be perceived as inferior due to biases from the
+larger languages' communites. In addition, communities for DSLs are generally 
+much smaller and can make DSLs seem weak socially.
```

Comment:
You mostly touched on size, but do you think there are other reasons DSLs might engender biases (either within a domain or across communities)? For example, do you think users of technology X would say things like "oh, people who work in field A use DSL Y, which is based on Ruby and is a girly language"?